Tests/GRPCTests/GRPCAsyncServerHandlerTests.swift

```diff
@@ -230,34 +230,6 @@ class AsyncServerHandlerTests: GRPCTestCase {
     try await responseStream.next().assertNil()
   }
 
-  func testResponseHeadersDroppedIfSetAfterFirstResponse() async throws {
-    throw XCTSkip("Setting metadata is racy. This test will not reliably pass until that is fixed.")
```

Comment:
This was removed because it no longer applies: we now trap (on debug builds) if headers are set after writing the first response